scripts/test_updates.sh

```diff
@@ -30,39 +33,43 @@ docker rm -f timescaledb-orig timescaledb-updated timescaledb-clean || true
 rm -rf  ${CLEAN_DATA_DIR} ${UPDATED_DATA_DIR}
```

Comment:
CLEAN_DATA_DIR and UPDATED_DATA_DIR is not used any more right ? One other place in the file as well.